src/types/Account/Account.d.ts

```diff
@@ -26,8 +26,6 @@ export declare class Account {
     getUnconfirmedBalance(displayDuffs?:boolean): number;
     getBIP44Path(network?:Network, index?:number): string;
 
-    getDPA(name: string): object;
-
     getIdentityHDKey(index: number, identityType: string): HDPrivateKey;
```

Comment:
Still Identity type?

Reply:
Oh ! Thank you, I've missed this one X.X